include/seqan3/alignment/aligned_sequence/aligned_sequence_concept.hpp

```diff
@@ -29,22 +29,21 @@ namespace seqan3::detail
 {
 
 //!\brief Helper function to deduce the unaligned sequence type from an aligned sequence container.
-template <template <typename ...> typename container_type, typename seq_alph_t, typename ...rest_t>
-//!\cond
+template <template <typename...> typename container_type, typename seq_alph_t, typename... rest_t>
     requires container<container_type<gapped<seq_alph_t>, rest_t...>>
-//!\endcond
 constexpr auto remove_gap_from_value_type(container_type<gapped<seq_alph_t>, rest_t...>)
     -> container_type<seq_alph_t, rest_t...>;
 
 //!\overload
-template <template <typename ...> typename container_type,
-          template <typename ...> typename allocator_type,
-          typename seq_alph_t, typename ...rest_t>
-//!\cond
+template <template <typename...> typename container_type,
+          template <typename...>
+          typename allocator_type,
```

Comment:
the first template class is on one line, the second is not?

Reply:
🤷